DiscreetORM.ts

```diff
@@ -116,6 +115,25 @@ export function WriteToDB(discreet_sql_io : DiscreetORMIO){
         return descriptor;
     }
 }
+export function WriteModifiedToDB(discreet_sql_io : DiscreetORMIO){
+    return function (target: Object, propertyKey: string | symbol, descriptor: PropertyDescriptor) {
+        const original_function = descriptor.value;
```

Comment:
I would add some documentation explaining how this works and how it is meant to be used

Reply:
done